src/api/app/components/workflow_run_filter_component.html.haml

```diff
@@ -21,37 +21,33 @@
     %h6.px-3.py-2
       %b Status
       = render partial: 'webui/shared/info_popover', locals: { position: 'bottom', text: status_info_text }
-    .px-4.py-1
+    .px-4
       = render partial: 'webui/shared/check_box', locals: { label: 'Succeeded', amount: @count['success'] || 0,
                                                             key: 'success', label_icon: 'fas fa-check text-primary',
                                                             checked: @selected_filter[:status]&.include?('success')}
-    .px-4.py-1
       = render partial: 'webui/shared/check_box', locals: { label: 'Running', amount: @count['running'] || 0,
                                                             key: 'running', label_icon: 'fas fa-running',
                                                             checked: @selected_filter[:status]&.include?('running')}
-    .px-4.py-1
       = render partial: 'webui/shared/check_box', locals: { label: 'Failed', amount: @count['fail'] || 0,
                                                             key: 'fail', label_icon: 'fas fa-exclamation-triangle text-danger',
                                                             checked: @selected_filter[:status]&.include?('fail')}
   .list-group.list-group-flush.mt-4.mb-4
```

Comment:
While we are here, we could get rid of traces of list-group classes, since that's not something we use anymore

Reply:
Good point, done!